ymca/src/main/webapp/WEB-INF/pages/includes/about.html

```diff
@@ -0,0 +1,113 @@
+<div class="container-fluid">
```

Comment:
this is already in a `container-fluid` in about.jsp. remove "container-fluid" class